### DISCUSSION TITLE
@gw8674's Pool (MyAZPool)

### DISCUSSION THREAD

**Discussion by gw8674** [tagyoureit/nodejs-poolController#720] [My Pool]
I’ve been meaning to post my specs here for a while and finally found some time this weekend to get on with it.

Just a plain and old (1987) rectangular pool with a PebbleTec & new tile facelift in 2007. Although, I’m pretty sure that I’m much prouder of what’s at the equipment pad and the electrons that make it all work rather than the pool/spa itself. ☺

20K gallon 19’x36’ Cantilevered Play/Lap Pool & 6' dia detached IG spa. Caribbean Blue Baja Mini Pebble.
<img width="669" alt="image" src="https://user-images.githubusercontent.com/52094163/232261911-5b9c2d15-740e-44cb-9f7c-e9750d07f82e.png">
<img width="788" alt="image" src="https://user-images.githubusercontent.com/52094163/232261928-86e523b5-30ab-40aa-b315-51aa725264a4.png">

Pentair IntelliCenter i10PS
Pentair IntelliFlo VSF Filter Pump
Pentair IntelliFlo3 VSF Spa Jet Pump
Pentair Clean & Clear 520 Filter
Pentair IntelliChlor IC-60
Pentair Chemical Acid Tank w/ Stenner Pump
(6) Pentair IntelliValves
Pentair MiniMax 100 Spa Heater
J&J Color Splash XG 120v LED’s
Maytronics Dolphin Active-60 Robotic Pool Cleaner
Solar Breeze Ariel Surface Robotic Cleaner
SP Auto Fill Leveler

The important stuff….
(1) Raspberry Pi 4B at the house running nodejs poolController and dashPanel.
(1) Raspberry Pi 4B in the enclosure running REM Chem Chemistry Controller. Stack includes:
     (2) Whitebox Labs Tentacle T3’s
     (1) Sequent Microsystems Industrial I2C Automation HAT
     (1) Sequent Microsystems 4-Relay HAT
     (1) Custom I/O board and I2C Extender/Active Terminator HAT
(1) Raspberry Pi 4B at the house running InfluxDB v1.8. Running Grafana locally and using VNC Viewer to access the RPi's.

Altelix 24x20x9 IP65 DIN Rail Fiberglass Enclosure w/ thermostatically controlled enclosure fans & 10” Touchscreen.
Atlas Scientific Industrial pH, ORP & EC probes.
Atlas Scientific Filter Pressure Transducer, OAT & HUM sensors.
FlowVis Digital Flow Meter
Precision Digital ProVu Pulse Input Flow Rate/Totalizer connected to automatic pool fill/leveler
<img width="603" alt="image" src="https://user-images.githubusercontent.com/52094163/232261978-9d3a1d76-7b18-4b02-9927-4a5a0d0e532c.png">
<img width="600" alt="image" src="https://user-images.githubusercontent.com/52094163/232262005-c689e938-af23-4af3-b835-1e077bceba15.png">

<img width="684" alt="image" src="https://user-images.githubusercontent.com/52094163/232262016-9d02bc51-5a27-422b-9185-f5eda2a06586.png">

<img width="685" alt="image" src="https://user-images.githubusercontent.com/52094163/232262031-9b10e3d0-c526-4514-8129-4994aa04ee26.png">

<img width="604" alt="image" src="https://user-images.githubusercontent.com/52094163/232262049-ce8ad6ba-20f3-48c0-93de-c299d4b675ef.png">

<img width="684" alt="image" src="https://user-images.githubusercontent.com/52094163/232262067-176fdb7d-0caa-4c08-aae0-cb96a39b3d19.png">

<img width="683" alt="image" src="https://user-images.githubusercontent.com/52094163/232262087-8ce063b2-5da9-44c8-911d-c894351e5e77.png">

<img width="686" alt="image" src="https://user-images.githubusercontent.com/52094163/232262104-cebba955-019b-4218-8d76-6ffc8bddddbe.png">

Thanks very much to rstrouse, tagyoureit and a few others along the way for all of the tremendous help and guidance during the last few years.

**Comment by rstrouse** [comment]
It is amazing how well all this works.  Helping another neighbor to get his up and running this weekend.  Congrats on your smart pool.

**Reply by gw8674** [reply]
Thanks @rstrouse ...  Actually I posted this back in July of last year... These are photos from back in 2020 mostly when we were still knee-deep into the dev phase.  However, I just happened to be checking out the "My Pools" section yesterday and noticed that the post like this one and one other in the REM section that I had made, were missing.  They were still in **my** "Your Discussions" area, but were un-viewable from the discussions areas.

I seem to be the latest victim of "Github" Gremlins 😁  Certainly not the first time they have got me.  Oh well... njspC, dashPanel and REM just continue to chug along like they are very happy campers...
Thanks again...

**Reply by tagyoureit** [reply]
Great setup!  Nice job putting it all together.

**Reply by truvec** [reply]
Definitely a benchmark. Funny when I first saw this post I thought " wow this guy really went all out to replicate MyAZPool's setup".
LOL

Thanks for all your work and documentation.

**Comment by Exit2Studios** [comment]
Any issues controlling the intelliflo3 pump?  I haven’t found anything that confirms support of the newer pumps.

**Reply by rstrouse** [reply]
Just installed one last weekend.  It works like a champ.  It is an Intelliflo. VSF.

**Comment by Exit2Studios** [comment]
Why are they cheaper than the older Intelliflos?  I have a Frankenstein Jandy dry end and Intelliflo wet end since my VSF control panel got fried, but can’t control it from njspc.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by rstrouse** [reply]
I can only guess that it has to do with their manufacturing costs or supply chain.  At one point IntelliFlo VSF was 800 bones.
